metaflow/plugins/aws/batch/batch.py

```diff
@@ -126,6 +126,15 @@ def launch_job(
                     'Unable to launch Batch job. No job queue '
                     ' specified and no valid & enabled queue found.'
                 )
+
+        """
```

Comment:
For this comment, I propose something like this (very similar to yours):
The environment variable for METAFLOW_DATASTORE_SYSROOT_LOCAL is *NOT* set because the syncing of the
metadata (provided in task_finished in batch_decorator.py and in _sync_metadata in batch_cli.py) assumes that DATASTORE_LOCAL_DIR is where the metadata is stored on the remote batch instance. This is the default value if METAFLOW_DATASTORE_SYSROOT_LOCAL is *NOT* set; if it is, the resulting path is different (see get_datastore_root_from_config in datastore/local.py)

Reply:
@valayDave can you lint your comment to 80 chars/line?